apps/hannk/interpreter/model.h

```diff
@@ -20,11 +20,6 @@ namespace hannk {
 class Op;
 using OpPtr = std::unique_ptr<Op>;
 
-template<class T, class... Args>
-std::unique_ptr<T> make_op(Args &&...args) {
-    return std::unique_ptr<T>(new T(std::forward<Args>(args)...));
```

Comment:
I was thinking this was good to keep to go with OpPtr. It enables easily changing to shared_ptr if needed (possible...).

Reply:
Fair enough, I'll revert that part but change the body of make_op to use make_unique